### DISCUSSION TITLE
Welcome to DDH-Community-Data-Wiki Discussions!

### DISCUSSION THREAD

**Discussion by aecryan** [delft-dh/DDH-Community-Data-Wiki#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jurra** [comment]
Hello everyone,
My name is Jose Carlos Urra @jurra . Research Software Engineer collaborating with Carola Hein to transfer know-how and best practices of FAIR data and software. Also helped together with @aecryan  in putting together an introductory workshop for Python in the context of GIS: https://the-magnificents.github.io/04-02-2021-Carpentry-for-HGIS/README.html.

**Comment by aecryan** [comment]
Hi all! My name is Ashley Cryan, @aecryan. I am a Data Manager with the TU Delft Digital Competence Centre and the Research Data Services team at the TU Delft Library. I've been working together with Jose, Carola, Yvonne, and Thomas for the last 6 months to support data literacy and building programming skills, and FAIR data infrastructure for the Delft Digital Humanities community.
